R/get_pro.R

```diff
@@ -5,77 +5,96 @@
 #' #'
 #' #' \describe{
 #' #'   \item{get_pro_groups}{Get the current meetup members from a pro meetup group}
-#' #'   \item{get_pro_events}{Get pro group events for the enxt 30 days}
+#' #'   \item{get_pro_events}{Get pro group events}
 #' #' }
 #' #'
 #' #' @template urlname
-#' #' @template verbose
 #' #'
 #' #' @references
-#' #' \url{https://www.meetup.com/meetup_api/docs/pro/:urlname/groups/}
-#' #' \url{https://www.meetup.com/meetup_api/docs/:urlname/events/#list}
+#' #' \url{https://www.meetup.com/api/schema/#ProNetwork}
 #' #'
 #' #' @examples
-#' #' \dontrun{
+#' #' try(
```

Comment:
```suggestion
#' #' \dontrun(
```

needs to be wraped in dontrun for when we submit to CRAN in the future